packages/augur-ui/src/modules/reporting/common.styles.less

```diff
@@ -1,5 +1,61 @@
 @import (reference) '~assets/styles/shared';
 
+.closeableNotice {
+  margin: @size-16 0;
+
+  > div {
+    align-items: center;
+    background: @color-closeable-notice-background;
+    border: 1px solid @color-outcome-two;
+    border-left-width: 0.25rem;
+    border-radius: @size-6;
+    display: flex;
+    font-family: Roboto Mono;
```

Comment:
can you use .mono-10-bold; for this style and the rest of the font styling parts here?

Reply:
Sure thing, I'll update this.